Adyen/adyen-magento2
# [PW-2386] Rate limiting notifications when authentication fails

**Description**
Add exponential rate limiting on the notification endpoint if username/password validation fails. Create a custom cache type with a specific tag and type identifier for this type of cache. 

**Tested scenarios**
- send notification to endpoint with correct username pwd, validate if request is accepted and not blocked after 6 tries
- send notification to endpoint with wrong username pwd and see if request is not rejected immediately
- send notification to endpoint with incorrect username pwd, validate if request is rejected while blocked after 6 tries
- send notification to endpoint with wrong username pwd while logging the cache key to verify the expiration date increases exponentially
